app/views/hyrax/base/unavailable.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>
-    <%= @presenter %> <%= @presenter.workflow.badge %>
+    <%= @presenter %> <%= @presenter.workflow.badge if @presenter.respond_to?(:workflow) && @presenter.workflow.present? %>
```

Comment:
When does `@presenter` not respond to `#workflow`?

It might be nicer just to ensure that `#workflow` and `#badge` produce reasonable defaults in the null case. Otherwise, perhaps `@presenter.try(:workflow).try(:badge)` would be more concise and easier to read.

Reply:
Did the presenter try try as suggested.